webrtc-stats.html

```diff
@@ -1128,9 +1129,18 @@
           <dfn>RTCTransportStats</dfn> dictionary
         </h3>
         <p>
-          A Transport carries a part of an SDP session, consisting of RTP and RTCP. When Bundle is
-          in use, an SDP session will have only one Transport per Bundle group. When Bundle is not
-          in use, there is one Transport per m-line.
+          An <code>RTCTransportStats</code> object represents the stats
+          corresponding to an <code><a>RTCDtlsTransport</a></code> and its
+          underlying <code><a>RTCIceTransport</a></code>. When RTCP multiplexing
+          is used, one transport is used for both RTP and RTCP. Otherwise, RTP
+          and RTCP will be sent on separate transports, and
+          <code>rtcpTransportStatsId</code> can be used to pair the resulting
+          <code>RTCTransportStats</code> objects. Additionally, when bundling
+          is used, a single transport will be used for all
+          <code><a>MediaStreamTrack</a></code>s in the bundle group. If
+          bundling is not used, different <code><a>MediaStreamTrack</a></code>
+          will use different transports. RTCP multiplexing and bundling are
+          described in [[!WEBRTC]].
```

Comment:
Since there is DtlsTransport and IceTransport, I am not sure if it is clear to a novice reader which "transports" are changing or multiplexed.

Do we need to clarify that somewhere?